index.html

```diff
@@ -4,197 +4,247 @@
+        </nav>
+    </header>
 
-                    </ul>
-        </section>
-        <section class="work-section">
-            <div class="work">
-                <img src="image/Snapshoot Portfolio.png" alt="tonic">
-                <h2 class="tonic">Tonic</h2>
-                <div class="works">
-                    <div class="canopy">CANOPY</div>
-                    <div class="dot"></div>
-                    <div class="back-end">Back End Dev</div>
-                    <div class="dot"></div>
-                    <div class="year">2015</div>
-                </div>
-                <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <ul class="skills">
-                <li>HTML</li>
-                <li>CSS</li>
-                <li>Javascript</li>
-                </ul>
-                <button class="project-button">See project</button>
+    <section class="header-ambony">
+        <div class="header-container">
+        <h1 class="narson">I'm Devesh.
+            <br>Glad to see you!
+        </h1>
+        <p class="detail">
+            I'm a software developer! I can 
+            help you build a product , feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <a href="#" class="connection"><h4>LET'S CONNECT</h4></a>
+
+        <ul class="social">
+            <li class="twitter">
+                <a href="https://twitter.com/@Devesh21148">
+                <img src="image/twitter.svg" alt="twitter" title="twitter"/>
+                </a>
+            </li>
+            <li class="linkindin">
+                <a href="https://www.linkedin.com/in/devesh-srivastava-7ab450216">
+                    <img src="image/Linkedin icon.svg" alt="Linkindin" title="linkindin"/>
+                </a>
+            </li>
+            <li class="github">
+                <a href="https://github.com/Devesh123987https://twitter.com/@Devesh21148">
+                <img src="image/github.svg" alt="github" title="github" />
+                </a>
+            </li>
+            <li class="angellist"><a href="#">
+                </a>
+                <img src="image/angellist.svg" alt="Angellist" title="Angellist" />
+                </a>
+            </li>
+            <li class="medium">
+                <a href="#">
+                <img src="image/medium.svg" alt="Medium" title="Medium" />
+                </a>
+            </li>
+        </ul>
+    </div>
+
+    </section>
+
+    <section class="work">
+        <div class="card-container">
+            <img class="sary" src="image/Snapshoot-Portfolio-desktop.svg" alt="Project Snapshoot" />
+            
+            <div class="work-fonony">
+
+            <h2 class="card-tittle">Tonic</h2>
+            <div class="frame2-img">
+                <img src="image/Frame 2.svg" />
```

Comment:
- [ ] [OPTIONAL] Please use the proper HTML semantics to display the short info of each project below the project heading this should be actual text content and not an image as this will ensure search engines and other assistive technologies can read it which will help the ranking of your website.
    -  Hint: you can use `ul` tag and wrap each piece of info (Facebook, Full stack dev, ...) in a `li` tag
 - This comment applies to all the info sections below the heading of each project.
     - Figma design:
       ![image](https://user-images.githubusercontent.com/104968236/230617793-ad30a91c-4522-4870-bc31-a669873e9c92.png)
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/230617882-16251ae6-2372-477f-975c-f31048c88090.png)